common/build/eslint-config-fluid/src/custom-rules/no-release-tags.js

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
I think we need to be a bit more specific in the rule name here. Since we're specifically targeting release tags on members of classes, types, and interfaces, something like `no-member-release-tags` would probably be fine.